modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/vulcan/openapi/contributor/APIApplicationOpenApiContributor.java

```diff
@@ -73,6 +74,10 @@ public void contribute(OpenAPI openAPI, OpenAPIContext openAPIContext)
 			return;
 		}
 
+		if (!FeatureFlagManagerUtil.isEnabled("LPD-35944")) {
```

Comment:
This code is in the Headless Builder module. This is not related to the Headless Builder (also called API Builder), so this code should not be here.

If there are no OpenAPIContributors in the Batch modules, you must create one